packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -37,10 +37,16 @@ class CreateCommand extends FlutterCommand {
       help: 'Also add a flutter_driver dependency and generate a sample \'flutter drive\' test.'
     );
     argParser.addFlag(
-      'plugin',
+      'package',
```

Comment:
I'd prefer `flutter create package` over `flutter create --template=package`, if we can make this work with the argument parser (ideally defaulting to 'application' if a sub-command to create isn't specified).

I personally think having the shorter flags is cleaner and easier to use than the longer `--template=...`. Note that we have mutually exclusive flags elsewhere, too -- the build mode flags, for example.